hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Mumble 1.4.0 Snapshot Announcement
+author: Robert Adam
+date: 2020-12-24
+categories:
+  - News
+  - Snapshot
+---
+
+This year we have a Christmas present for you: We are proud to announce that the first **snapshot release of Mumble 1.4.0** is released as of now. It
+is a very important step towards the actual release of version 1.4.0.
+
+After having understood what it means for a release to be a _snapshot release_ (see below), you can download the new version from our [**Downloads
+page**]({{< relref "/downloads" >}}).
+
+<!--more-->
+
+## Snapshot releases
+
+A _snapshot release_ can be considered an open-beta for an upcoming feature release. This means that it contains the latest and greatest features that
+were introduced since the last feature release. This does also mean that these features have not been tested by a broad audience yet and are
+therefore likely to still contain bugs. These could be minor ones, but in theory these could also be major. Therefore it is important that you always
+back up your data (e.g. configuration files) if you have a previous version of Mumble installed.
+
+Due to the reasoning stated above, we do not recommend using snapshots in an environment where stability of the software is key (e.g. in a business
+context).
+
+That being said though, we strongly encourage you to try snapshots out whenever possible and report any problems you encounter to our [**issue
+tracker**](https://github.com/mumble-voip/mumble/issues). This kind of feedback is very important for us to be able to make the actual release as
+stable and reliable a possible.
+
+
+## Feature spotlight
+
+As there are many, many changes with this new version, here is a selection of things that we think are most prominent. If you're interested in all
+changes, have a look at the full [changelog](#Changelog).
+
+### Channel listeners
+
+### TalkingUI
+
+### UI indication for access restrictions
+
+### Markdown
+
+### Stereo playback
+
+### Nicknames
+
+
+## Thanks
+
+We want to thank everyone who spent some time to work on the project. Without this kind of help from you, we would not be able to provide this
+software in the way that it is.
+
+
+## Changelog
```

Comment:
This is quite long, maybe it could be put somewhere else, so it doesn't make the blogpost too long.
Otherwise it could maybe be put in a [collapsible section](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) ... though lists inside a collapsible section don't work in some coverters, maybe hugo is different.

Reply:
If Hugo can cope with collapsible sections, that'd be a good idea. Moving this to somewhere else, I don't like since we don't have space for it on our download page and we'll not have a GitHub release for this where we could put this :eyes: 

@Kissaki do you happen to know whether hugo can create collapsible sections?

Reply:
Markdown does not support collapsible sections. But HTML can be embedded just fine. We could also introduce shortcodes for this. Both are fine solutions to me.

Noteworthy, we did use HTML with more structuring and collapsibles for the big 1.3 release notes.

So feel free to put `<details><summary>caption</summary>content</details>` into the markdown file. Or create (or ask me to create) a shortcode for it.

Reply:
I now made the changelog collapsible. Introducing shortcodes sounds like a good idea, even though I don't know how to do that (nor am i entirely sure of what exactly these are xD) :point_up: